src/menu/views/load_rom.c

```diff
@@ -206,8 +206,50 @@ static void load (menu_t *menu) {
 
     menu->next_mode = MENU_MODE_BOOT;
     menu->boot_params->device_type = BOOT_DEVICE_TYPE_ROM;
-    menu->boot_params->tv_type = BOOT_TV_TYPE_PASSTHROUGH;
     menu->boot_params->detect_cic_seed = true;
+    
+    if (menu->settings.autodetect_rom_region) {
+        // check the market type (menu->load.rom_info.destination_code) and set best guess?!
+        switch (menu->load.rom_info.destination_code) {
```

Comment:
Could you please move this switch to a function? It might be useful in the other places as well.

Reply:
Done.